Cargo.toml

```diff
@@ -91,7 +91,7 @@ cmake = "0.1.51"
 document-features = "0.2.10"
 hashbrown = { version = "0.14.5", default-features = false } # A faster hashmap, nostd compatible
 libc = "0.2.159" # For (*nix) libc
-libipt = "0.1.4"
+libipt = { git = "https://github.com/Marcondiro/libipt-rs" }
```

Comment:
We'll have to upstream the stuff or upload a fork to crates.io

Reply:
Yep, I'm just testing out windows in CI before publishing, I'll then mark this as ready and not draft :)

Reply:
sweet